cluster/gce/configure-vm.sh

```diff
@@ -529,6 +529,12 @@ function salt-set-apiserver() {
 EOF
 }
 
+function salt-set-test-node() {
```

Comment:
nit: can you call this salt-set-test-grain or something so it isn't misleading if we call it to set the same grain on the master?